1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,44 @@
 // Write your code here
+const items = document.querySelectorAll('img');
+const nextItem = document.querySelector('.next');
+const previousItem = document.querySelector('.previous');
+
+let count = 0;
```

Comment:
there is a comment at the bottom of the script about the name `count` - it seems it should be renamed to `index`
another potential bug here is `let` keyword here. i guess you want it to be a global variable. but `const` is for immutable variable. and `let` is more for local mutable variables. so the only better alternative left for declaring a global mutable variable is `var` keyword ;)

Reply:
just a second thought about it: 
you are allowed to use `let` for global mutable variables.
it might be a potential bug only in case if you decide to move it down to the bottom of the file. like in `Quote Generator` task, coz variable declared with `let` keyword can't be hoisted at the global scope.
more about it is in the below links:
- https://www.w3schools.com/js/js_hoisting.asp
- https://developer.mozilla.org/en-US/docs/Glossary/Hoisting